### PR TITLE
added new test to parent_label

### PR DESCRIPTION
**Pull request by amaarora**

### CODE REVIEW

**Comment by amaarora** [comment]
I wrote a description but not sure why it is not showing on the PR. 
I have added a new test to `parent_label`

`def parent_label(o, **kwargs):
    "Label `item` with the parent folder name."
    return o.parent.name if isinstance(o, Path) else o.split(os.path.sep)[-1]`

Currently, there is no test to check `else o.split(os.path.sep)[-1]`

ie.,  the only tests that exist for `parent_label` are 
`test_eq(parent_label(items[0]), '3')
[parent_label(o) for o in items]`

And items[0] is of type Path. 

I have added a new test for type `str` and also believe `else o.split(os.path.sep)[-1]` should be updated to `else o.split(os.path.sep)[-2]` to return parent label. 

Regards

**Comment by jph00** [comment]
Good idea! Note that you need to run `tools/run-after-git-clone`, re-export, and then push again, since you have some extra metadata in your diff.

**Comment by amaarora** [comment]
@jph00 Done :) 
Should be ok now - only 1 thing though, I had to manually update outputs of `untar_data` ie., absolute paths.

**Comment by jph00** [comment]
Great! Don't worry in the future about editing outputs like that - it's unlikely that would result in a conflict, whereas document metadata changes are very likely to result in conflicts.

File: dev/05_data_core.ipynb [hunk 1]
```diff
@@ -405,6 +405,7 @@
    ],
    "source": [
     "test_eq(parent_label(items[0]), '3')\n",
+    "test_eq(parent_label(\"fastai_dev/dev/data/mnist_tiny/train/3/9932.png\"), '3')\n",
```
Comment (kdorichev) [inline dev/05_data_core.ipynb]: test_eq(parent_label(str(items[0])), '3')

Comment (amaarora) [reply dev/05_data_core.ipynb]: Hi @kdorichev 
That definitely looks more concise to me

**Pull request merged**